### PR TITLE
Write test cases for schemaSampleSize

### PR DESCRIPTION
**Pull request by ykye**
Write test cases for schemaSampleSize

BugzID: 59432

@HolgerKache @mayya-sharipova

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ A utility is included to create test data to the cloudant database.
   ```
      python -m helpers.dataload -load 200
   ```
+- Run `python -m helpers.schema -addDocs` to add docs for schema test
```
Comment (HolgerKache) [inline test/README.md:62]: Please add this additional comment:

_Note_: To load custom documents, put them in folder `test/resources/<database>.json` where `<database>` represents the database to load the document into. An invalid database name will be ignored. Documents are supplied in a `docs` array for bulk loading, e.g.:

`test/resources/n_customer.json`

```
{
    "docs":[
        {
            "_id": "1",
            "name":{
                "firstname":"Jim",
                "lastname":"Yal"
            },
            "age":20,
            "total_miles":true
        },
     ...
    ]
}
```

**Comment by HolgerKache** [comment]
@ykye could you please take care of the error I posted here on Slack: https://clouddataservices.slack.com/archives/cloudant-spark/p1453742583000053

There is one additional comment on the README but everything else looks great. Especially the test cases are very complete and the new helpers.schema method works well. Thank you!

+1 after we figured out the error above.

**Comment by ykye** [comment]
@HolgerKache I have committed some new changes. Can you please take a look at if these updates have addressed all of your comments? Thanks!

**Comment by HolgerKache** [comment]
+1

**Pull request merged**